revolv/templates/project/post_funding_update.html

```diff
@@ -31,7 +31,7 @@
                     {% include "base/partials/tooltip_formfield.html" with field=form.solar_url %}
                 </div>
             </div>
-            <button class="large right" id="save_project" type="submit">Save</button>
+            <button class="large right" id="save_project" type="create">Save</button>
```

Comment:
Also it looks like these templates are exactly the same except for this line, so they should also be DRY'd up